shell/browser/electron_browser_main_parts.cc

```diff
@@ -439,6 +440,11 @@ int ElectronBrowserMainParts::PreMainMessageLoopRun() {
   SpellcheckServiceFactory::GetInstance();
 #endif
 
+#if BUILDFLAG(IS_WIN)
+  // access ui native theme here to prevent blocking calls later
+  base::win::AllowDarkModeForApp(true);
```

Comment:
@VerteDinde can we move this into the executable startup path https://github.com/electron/electron/blob/main/shell/app/electron_main_win.cc#L221 similar to Chrome / content_shell ?

Reply:
@deepak1556 Yeah I think that would work! I'll put in a PR to fix that 🙇‍♀️

Reply:
https://github.com/electron/electron/pull/35529